stage2/04-install-piscsi/01-run.sh

```diff
@@ -16,12 +16,12 @@ echo "CI is $CI"
 on_chroot << EOF
 su pi
 cd /home/pi/
-git clone https://github.com/akuker/RASCSI.git
-cd RASCSI
+git clone https://github.com/piscsi/piscsi.git
```

Comment:
Do you want to make this `https://github.com/PiSCSI/piscsi` to match the actual github group name? Both seem to work, so it might be optional though.

Reply:
Agreed. Updated